src/bundle/Resources/views/layout.html.twig

```diff
@@ -101,6 +101,15 @@
         %}
             <script type="text/javascript" src="{{ asset_url }}"></script>
         {% endjavascripts %}
+
+        {% javascripts
```

Comment:
Why we need another `<script>`? Cannot we combine this with the one above?

Reply:
Probably we can, but I think will be good to leave it as separate JS file. Message catalogs and config file for the `Translator` object are generated as static files by `php bin/console bazinga:js-translation:dump ...` command, so IMHO no fits to our or 3rd party javascript files.

Reply:
Nevermind.